content/departments/engineering/dev/cross-product/code-intelligence-platform/process-and-timeline.md

```diff
@@ -0,0 +1,144 @@
+# Code Intelligence Platform strategy planning process and timeline
+
+Note that this won’t be the top priority of product nor design until after sept 27 ([Strategic-readiness](../strategic/index.md) work comes first).
+
+> Important tactical note: further specifics for each stage will be added on a rolling basis.
+
+<object data="process-roadmap-high-level.svg" width="835" height="327" aria-label="Diagram showing a single timeline, with the first section title from left to right being vision, the second strategy, and the third and roadmap. The text across all 3 sections reads as follows, from left to right: hypotheses, validation, definition, top level strategy, tactical strategy, team/feature roadmap."></object>
```

Comment:
Does this SVG come from Figma? If yes, you don't need to repeat the text in the ARIA label if you use the Export panel and untick the option to outline text (makes the SVG smaller too). Just don't use any exotic fonts (e.g. you can use the Source Code font from the brand guidelines, which is used in the handbook too)

<img width="495" alt="Screen Shot 2022-09-16 at 19 20 27" src="https://user-images.githubusercontent.com/10532611/190836802-d111edf9-ef8c-4288-b107-c75b80ec3178.png">

Reply:
ah, thanks! That's super useful to know. (Yes, of course the svg comes from figma :) ).